hnn_core/cell_response.py

```diff
@@ -456,5 +456,4 @@ def read_spikes(fname, gid_ranges=None):
     if gid_ranges is not None:
```

Comment:
```suggestion
    # Handle edge case where cell never spikes.
    # Otherwise it would not appear when written to a file.
    if gid_ranges is not None:
```

Reply:
This doesn't have to do with spiking. `gid_ranges=None` by default unless the user is required to define it when loading a legacy spike file that doesn't contain the 3rd column.

Reply:
Ah I see what you mean, I misunderstood the situation where this is raised